_posts/2025-05-18-update.md

```diff
@@ -7,6 +7,14 @@ slug: 2025-05-18-update
 
 ## Developer News
 
+James Sturtevant and Amim Knabben are stepping down from their roles as techincal leads in [SIG Windows](https://github.com/kubernetes/community/tree/master/sig-windows) and Yuanliang Zhang is notimated as the new Lead
+
+Wenjia Zhang has stepped down as the co-chair of Kubernetes [SIG etcd](https://github.com/kubernetes/community/blob/master/sig-etcd/README.md). [Siyuan Zhang](https://github.com/siyuanfoundation) is nominated to take over Wenjia's role as the co-chair.
+
+Kubernetes has released multiple minor versions including [v1.33.1](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.33.md), [1.32.5](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.32.md), [1.31.9](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.31.md) and [1.30.13](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.30.md).
```

Comment:
This can go in the release schedule. I can move it when cleaning up though. Noting for next time.